docs/content/kv_commands.md

```diff
@@ -53,16 +53,20 @@ $ wrangler kv:namespace rename f7b02e7fc70443149ac906dd81ec1791 "updated kv name
 Outputs a list of all KV namespaces associated with your account id.
 
 #### Usage
+The example below uses Python's JSON pretty-printing command line tool to pretty-print output.
 
 ```sh
-$ wrangler kv:namespace list
-🌀  Retrieving namespaces
-✨  Success:
-+------------------+----------------------------------+
-| TITLE            | ID                               |
-+------------------+----------------------------------+
-| new kv namespace | f7b02e7fc70443149ac906dd81ec1791 |
-+------------------+----------------------------------+
+$ wrangler kv:namespace list | python -m json.tool
```

Comment:
Why can't we pretty print it from Rust?

Reply:
Enough people have brought this up that I'll add pretty printing and a flag for it :)

Reply:
nvm, gonna recommend folks use jq. It's hard for us to pretty-print the keys json since it is streamed.